### PR TITLE
Villain-Con Project :millestone 1 

### PR DESCRIPTION
**Pull request by ambrose-kibet**
### In this Milestone we:
- Add desktop version for the project.
- Add 'Featured speakers' dynamically using javsacript.
- Add a README description.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank Microverse.org
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
```
Comment (ghost) [inline README.md]: - [x] as stated in the project requirement, Kindly provide appropriate credit to [Cindy Shin](https://www.behance.net/adagio07) the [author of the original design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), as required by the [Creative Commons License](https://creativecommons.org/licenses/) in your acknowledgement section

File: README.md [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+### Key Features <a name="key-features"></a>
+
+- **About**
+- **Home**
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](https://ambrose-kibet.github.io/VIllain-con-capstone-project/)
+- [Video Demo Link](https://www.loom.com/share/97e40c3ca1f245c998d470d90a986101)
```
Comment (ghost) [inline README.md:56]: - your live demo currently displays only the mobile version of your website. Kindly update it to display both mobile and desktop versions

File: index.html [hunk 3]
```diff
@@ -18,12 +18,20 @@
...
           <i class="fa-solid fa-bars"></i>
         </button>
+        <ul class="links-container horizontal-links">
+          <li><a href="./about.html" class="link active">About</a></li>
+          <li><a href="#" class="link">Program</a></li>
+          <li><a href="#" class="link">Join</a></li>
+          <li><a href="#" class="link">Sponsonr</a></li>
+          <li><a href="#" class="link">News</a></li>
+        </ul>
+        <a href="#" class="link link-btn large-nav">CC Campaign</a>
```
Comment (ghost) [inline index.html:53]: - [x] Kindly add the missing social-media links section to your nav bar to match the original design

image

File: index.html [hunk 4]
```diff
@@ -58,81 +71,84 @@ <h2 class="h1">2023.10.16 (THU) ~ 17(FRI)</h2>
       <section class="section-container program-section">
         <h2 class="section-title">Main-program</h2>
```
Comment (ghost) [inline index.html:91]: - [x] I would highly recommend that you add background images to your design to make it more appealing like in the original design

image


### Checkout original design

image

File: index.html [hunk 5]
```diff
@@ -58,81 +71,84 @@ <h2 class="h1">2023.10.16 (THU) ~ 17(FRI)</h2>
...
+            </span>
+            <p class="p-title">Lecture</p>
+            <p>
+              Lorem ipsum dolor sit amet consectetur adipisicing elit. Placeat
+              eveniet et corporis, corrupti sint fugit!.
+            </p>
+          </article>
+          <article class="program">
+            <span class="program-icon">
+              <img
```
Comment (ghost) [inline index.html]: - [x] Kindly provide some margin around the cards in the program section to match the original design. You can also implement the hover effect on the cards

image

### Checkout the original design

image

**Review by ghost — CHANGES_REQUESTED** [review]
Hi @ambrose-kibet , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights

✅ Correct use of Github workflow
✅ No linter error


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

N/A
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 2 above)
Comment (Favourezeugwa) [reply README.md:56]: [OPTIONAL]. 👍🏽 
- Just as @frank1738 suggested, kindly update your README file with an updated live demo link 🤗 

- But if the link you deployed is different from your `desktop` branch, then you can go ahead and deploy after it has been merged.

File: README.md (hunk 1 above)
Comment (Favourezeugwa) [reply README.md]: - I'd like to kindly suggest you update your README file with an appropriate credit to [Cindy Shin](https://www.behance.net/adagio07) the [author of the original design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), as required by the [Creative Commons License](https://creativecommons.org/licenses/) in your acknowledgement section 🌸

**Review by Favourezeugwa — CHANGES_REQUESTED** [review]
## STATUS: REQUIRES CHANGES 🚦 ⚠️ ♻️ 

Hi @ambrose-kibet , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

so-close-brianna


## Highlights 💯 
- No Linter errors ✔️
- Descriptive Pull request summary✔️
- Good commit messages ✔️
- Professional Readme file ✔️ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by levy002 — APPROVED** [review]
## Hello @ambrose-kibet 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Status: APPROVED :fire:  :fire: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

- [OPTIONAL] Kindly make sure to fix the deployment as suggested as the previous reviwer.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**